lib/clients/helm.js

```diff
@@ -504,6 +504,9 @@ class Helm {
     }
     return this.kubectl.cmd(`create namespace ${ns}`);
   }
+  _deploymentName(){
```

Comment:
I think is fine, I also checked that that config exits:
https://github.com/w3f/polkadot-deployer/blob/f49c75a478241a1208dc428a7537fe47a81e4b2f/lib/cluster/strategies/remote/index.js#L37-L39